perllib/FixMyStreet/Cobrand/Hackney.pm

```diff
@@ -59,6 +61,41 @@ sub geocoder_munge_results {
     $result->{display_name} =~ s/, London Borough of Hackney//;
 }
 
+sub addresses_for_postcode {
+    my ($self, $postcode) = @_;
+
+    my $api = $self->feature('address_api');
+    my $url = $api->{url};
+    my $key = $api->{key};
+
+    $url .= '?format=detailed&postcode=' . uri_escape_utf8($postcode);
+    my $ua = LWP::UserAgent->new;
+    $ua->default_header(x_api_key => $key);
+
+    my $pages = 1;
+    my @addresses;
+    for (my $page = 1; $page <= $pages; $page++) {
+        my $res = $ua->get($url . '&page=' . $page);
+        my $data = decode_json($res->decoded_content);
+        $pages = $data->{data}->{page_count};
+        foreach my $address (@{$data->{data}->{address}}) {
+            next unless $address->{locality} eq 'HACKNEY';
```

Comment:
I wonder if there's something to be said for flagging when the postcode is outside hackney. Rather than just saying "we can't find the address"